code/modules/admin/admin_investigate.dm

```diff
@@ -1,22 +1,48 @@
+#define INVESTIGATE_NOTES "notes, memos, watchlist" // Not a real investigation category
+#define INVESTIGATE_FILE(subject) file("[GLOB.log_directory]/[subject].html")

```

Comment:
I feel these defines aren't actually helpful and just bloat. They're local to the file and cover a whole 3 lines.

Reply:
Should I remove the defines, or just undef them at the eof?

Reply:
Just remove.